html-templates/cbl/missing-ratings.tpl

```diff
@@ -6,7 +6,7 @@
     {load_templates "subtemplates/paging.tpl"}
 
     <header class="page-header">
-        <h2 class="header-title">Missed Ratings</h2>
+        <h2 class="header-title">{tif $student != $.User ? $student->FirstNamePossessive} Missed Ratings</h2>
```

Comment:
We should probably do this check on the ID to make sure it works correctly even if caching doesn't ensure they're the same model instance

```suggestion
        <h2 class="header-title">{tif $student->ID != $.User->ID ? $student->FirstNamePossessive} Missed Ratings</h2>
```